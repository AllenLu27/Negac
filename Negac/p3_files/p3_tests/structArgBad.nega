void fn(struct a b) {
}
