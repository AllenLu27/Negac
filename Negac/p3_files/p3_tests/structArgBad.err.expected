FATAL [1,9]: Unknown struct used
Semantic analysis failed
